Misc/NEWS.d/next/Library/2024-08-18-08-25-32.gh-issue-123084.rf8izX.rst

```diff
@@ -0,0 +1,2 @@
+:class:`!shutil.ExecError` is now a deprecated alias of
+:class:`RuntimeError`.
```

Comment:
```suggestion
Deprecate :class:`!shutil.ExecError`, which hasn't been
raised by any :mod:`shutil` function since Python 3.4. It's
now an alias for :exc:`RuntimeError`.
```